docs/sdk/src/reference_docs/frame_currency.rs

```diff
@@ -1,8 +1,99 @@
 //! FRAME Currency Abstractions and Traits
 //!
-//! Notes:
+//! TODO:
 //!
-//! - History, `Currency` trait.
-//! - `Hold` and `Freeze` with diagram.
-//! - `HoldReason` and `FreezeReason`
-//! 	- This footgun: https://github.com/paritytech/polkadot-sdk/pull/1900#discussion_r1363783609
+//! - [x] History, `Currency` trait.
+//! - [ ] `Hold` and `Freeze` with diagram.
+//! - [x] `HoldReason` and `FreezeReason`
+//! - [ ] This footgun: https://github.com/paritytech/polkadot-sdk/pull/1900#discussion_r1363783609
```

Comment:
I don't think we should include discussions in the ref docs, as from my point of view this section would get obsolete as soon as the discussion reached an agreement, in which case we could document the output. But I am open to arguments

Reply:
This is a fungible / fungibles thing, should probably be documented in the fungible docs?